tests/services/test_service.py

```diff
@@ -136,3 +136,22 @@ def test_missing_or_invalid_type(lang_data2, service, identity):
         lang_data2['props']['newkey'] = v
         pytest.raises(
             ValidationError, service.create, identity, lang_data2)
+
+
+def test_read_all_no_cache(lang_type, lic_type, lang_data_many, service, identity):
```

Comment:
Since cache/no cache is checked at records-resources level... I think is safe to just tests that we only get vocabularies of one type.
```suggestion
def test_read_all(lang_type, lic_type, lang_data_many, service, identity):
```

Reply:
still, we can write a basic test if it works for vocab so I left it

Reply:
Sounds good :)